### PR TITLE
Docs: fix auth and expand appearance

### PR DESCRIPTION
**Pull request by vdboor**
Fixed the appearance of `frozenset({...})` in the HTML, and also brushed up the expand/embed appearance.

### CODE REVIEW

**Review by lbam — APPROVED** [review]

**Review by jjmurre — APPROVED** [review]
Ziet er goed uit!

File: docs/source/datasets.py [hunk 1]
```diff
@@ -153,45 +150,73 @@ def render_wfs_dataset_docs(dataset: DatasetSchema, dataset_path: str):
...
+        "auth": table.auth | table.dataset.auth,
+        "expands": _get_table_expands(table),
         "additional_filters": table.additional_filters,
-        "additional_relations": table.additional_relations,
         "source": table,
         "has_geometry": _has_geometry(table),
     }
 
 
+def _get_table_uri(table: DatasetTableSchema, paths: dict[str, str]) -> str:
```
Comment (gkoller) [inline docs/source/datasets.py:181]:
```python 
from more_ds.network import URL
...
BASE_URL = URL("https://api.data.amsterdam.nl")
...
def _get_table_uri(table: DatasetTableSchema, paths: dict[str, str]) -> URL:
   ...
```

Comment (gkoller) [reply docs/source/datasets.py:181]: Make it clearer what kind string you are actually returning

File: docs/source/datasets.py [hunk 2]
```diff
@@ -153,45 +150,73 @@ def render_wfs_dataset_docs(dataset: DatasetSchema, dataset_path: str):
...
+    """Tell where the endpoint of a table will be"""
+    dataset_path = paths[table.dataset.id]
+    snake_id = to_snake_case(table.id)
+    return f"{BASE_URL}/v1/{dataset_path}/{snake_id}/"
```
Comment (gkoller) [inline docs/source/datasets.py:185]: When you use `URL`, URL constructions becomes succint:
```python
return BASE_URL / "v1" / dataset_path / snake_id
```

File: docs/source/datasets.py [hunk 3]
```diff
@@ -153,45 +150,73 @@ def render_wfs_dataset_docs(dataset: DatasetSchema, dataset_path: str):
...
+                "id": additional_relation.id,
+                "camel_name": toCamelCase(additional_relation.id),
+                "snake_name": to_snake_case(additional_relation.id),
+                "relation_id": additional_relation.relation.replace(":", rel_id_separator),
+                "target_doc_id": f"{related_table.dataset.id}{rel_id_separator}{related_table.id}",
+                "related_table": related_table,
+            }
+        )
+
+    return sorted(expands, key=lambda item: item["id"])
```
Comment (gkoller) [inline docs/source/datasets.py:217]:
```python
return sorted(expands, key=operator.itemgetter("id"))
```

File: docs/source/datasets.py [hunk 4]
```diff
@@ -200,42 +225,37 @@ def _has_geometry(table: DatasetTableSchema) -> bool:
...
 
 
-def _get_feature_type_context(table: DatasetTableSchema, parent_path: str):
+def _get_feature_type_context(table: DatasetTableSchema, paths: dict[str, str]):
     """Collect all table data for the WFS server spec."""
     snake_name = to_snake_case(table.dataset.id)
     snake_id = to_snake_case(table["id"])
-    uri = f"{BASE_URL}/v1/{parent_path}/{snake_id}/"
+    parent_path = paths[table.dataset.id]
+    uri = f"{BASE_URL}/v1/wfs/{parent_path}/{snake_id}/"
```
Comment (gkoller) [inline docs/source/datasets.py:233]: When using `URL`:
```python
uri = BASE_URL / "v1/wfs" / parent_path / snake_id
```

File: docs/source/datasets.py [hunk 5]
```diff
@@ -200,42 +225,37 @@ def _has_geometry(table: DatasetTableSchema) -> bool:
...
-        ],
+        "expands": _get_table_expands(table, rel_id_separator="/"),
         "source": table,
         "has_geometry": has_geometry,
         "wfs_typename": f"app:{snake_name}",
         "wfs_csv": (
-            f"{BASE_URL}/v1/wfs/{parent_path}/?SERVICE=WFS&VERSION=2.0.0"
-            f"&REQUEST=GetFeature&TYPENAMES={snake_id}&OUTPUTFORMAT=csv"
+            f"{uri}?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature"
+            f"&TYPENAMES={snake_id}&OUTPUTFORMAT=csv"
```
Comment (gkoller) [inline docs/source/datasets.py:252]: When using `URL`:

```python
uri // dict(SERVICE="WFS",
            VERSION="2.0.0",
            REQUEST="GetFeature",
            TYPENAMES=snake_id,
            OUTPUTFORMAT="csv")
```

Comment (gkoller) [reply docs/source/datasets.py:252]: Arguably more readable.

File: docs/source/datasets.py [hunk 6]
```diff
@@ -200,42 +225,37 @@ def _has_geometry(table: DatasetTableSchema) -> bool:
...
             if has_geometry
             else ""
         ),
         "wfs_geojson": (
-            f"{BASE_URL}/v1/wfs/{parent_path}/?SERVICE=WFS&VERSION=2.0.0"
-            f"&REQUEST=GetFeature&TYPENAMES={snake_id}&OUTPUTFORMAT=geojson"
+            f"{uri}?SERVICE=WFS&VERSION=2.0.0&REQUEST=GetFeature"
+            f"&TYPENAMES={snake_id}&OUTPUTFORMAT=geojson"
```
Comment (gkoller) [inline docs/source/datasets.py:258]: See https://github.com/Amsterdam/dso-api/pull/365/files#r722297778

**Review by gkoller — APPROVED** [review]

**Pull request merged**